main/arbitration_identifiers.c

```diff
@@ -5,10 +5,6 @@ uint32_t read_uint32_be(uint8_t *data) {
   return data[0] | (data[1] << 8) | (data[2] << 16) | (data[3] << 24);
 }
 
-void update_send_identifier(uint32_t new_send_identifier) {
-  send_identifier = new_send_identifier;
-}
-
-void update_receive_identifier(uint32_t new_receive_identifier) {
-  receive_identifier = new_receive_identifier;
+uint32_t read_uint32_le(uint8_t *data) {
```

Comment:
this _le / _be might be wrong. 

little endian = LSB
big endian = MSB

do i have it right? also this could go to a utilities file name wise. there might be like a `bswap`ish function for this?